srcpkgs/libopenmpt/template

```diff
@@ -0,0 +1,85 @@
+# Template file for 'libopenmpt'
+pkgname=libopenmpt
+version=0.4.11
+revision=1
+wrksrc="libopenmpt-${version}+release.autotools"
+build_style=gnu-configure
+configure_args="--prefix=/usr
+	$(vopt_with zlib)
+	$(vopt_with mpg123)
+	$(vopt_with ogg) $(vopt_with vorbis) $(vopt_with vorbis vorbisfile)
+	$(vopt_with pulseaudio)
+	$(vopt_with portaudio) $(vopt_with portaudiocpp)
+	$(vopt_with sdl2) $(vopt_with sdl)
+	$(vopt_with sndfile)
+	$(vopt_with flac)
+	$(vopt_enable libopenmpt_modplug) $(vopt_enable libmodplug)
+	$(vopt_enable openmpt123)"
+hostmakedepends="pkg-config"
+makedepends="
+	$(vopt_if zlib zlib-devel)
+	$(vopt_if mpg123 mpg123-devel)
+	$(vopt_if ogg libogg-devel) $(vopt_if vorbis libvorbis-devel)
+	$(vopt_if pulseaudio pulseaudio-devel)
+	$(vopt_if portaudio portaudio-devel) $(vopt_if portaudiocpp portaudio-cpp-devel)
+	$(vopt_if sdl2 SDL2-devel) $(vopt_if sdl SDL-devel)
+	$(vopt_if sndfile libsndfile-devel)
+	$(vopt_if flac libflac-devel)"
+depends="
+	$(vopt_if zlib zlib)
+	$(vopt_if mpg123 libmpg123)
+	$(vopt_if ogg libogg) $(vopt_if vorbis libvorbis)
+	$(vopt_if pulseaudio libpulseaudio)
+	$(vopt_if portaudio portaudio) $(vopt_if portaudiocpp portaudio-cpp)
+	$(vopt_if sdl2 SDL2) $(vopt_if sdl SDL)
+	$(vopt_if sndfile libsndfile)
+	$(vopt_if flac libflac)"
+short_desc="Cross-platform C & C++ library to decode tracked music files (modules)"
+maintainer="a dinosaur <nick@a-dinosaur.com>"
+license="BSD-3-Clause"
+homepage="https://lib.openmpt.org/libopenmpt/"
+distfiles="https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-${version}+release.autotools.tar.gz"
+checksum=260e92cc2f6af37113442bff2c75a3c36a09eba4078dc593203a0502f95d26bd
+conflicts="$(vopt_if libmodplug libmodplug)"
+
+post_install() {
+	vlicense LICENSE
+}
+
+# Package build options
+build_options="
+	zlib
+	mpg123
+	ogg vorbis
+	pulseaudio
+	portaudio portaudiocpp
+	sdl2 sdl
+	sndfile
+	flac
+	libopenmpt_modplug libmodplug
+	openmpt123"
+build_options_default="zlib mpg123 ogg vorbis pulseaudio portaudio portaudiocpp sndfile flac openmpt123"
+vopt_conflict sdl2 sdl
+
+libopenmpt-devel_package() {
+	depends="${sourcepkg}-${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/share/doc/libopenmpt/examples
+		vmove usr/include
+		vmove usr/lib/pkgconfig
+		vmove "usr/lib/*.a"
+		vmove "usr/lib/*.so"
+	}
+}
+
+if [ ! -z "$build_option_openmpt123" ]; then
```

Comment:
I really don't see the use case for not building a sub-package that could be built in any case. There's no technical reason to not build it and it is already split off so if you don't want the binary, you could just throw the package away or not install it. I would just remove that conditional and the build option.

Reply:
Pretty silly in hindsight, fixed.